src/main/java/org/folio/circulation/resources/RegularCheckOutStrategy.java

```diff
@@ -41,10 +44,17 @@
 
   private Result<LoanAndRelatedRecords> refuseWhenItemIsNotLoanable(LoanAndRelatedRecords relatedRecords) {
     final Loan loan = relatedRecords.getLoan();
+    final LoanPolicy loanPolicy = loan.getLoanPolicy();
 
-    if (loan.getLoanPolicy().isNotLoanable()) {
+    if (loanPolicy.isNotLoanable()) {
       String itemBarcode = loan.getItem().getBarcode();
-      return failed(singleValidationError("Item is not loanable", ITEM_BARCODE, itemBarcode));
+
+      Map<String, String> parameters = new HashMap<>();
```

Comment:
I might be tempted to extract this into a method like `loanPolicyValidationError(loanPolicy, message, additionalParameters)`

And then replace usage of `LoanPolicy.errorForPolicy` with this.

Reply:
Makes sense for me. Do you think `LoanPolicy` class is an appropriate place for this?

Reply:
Probably at this point, I'd prefer it wasn't but that is likely a bigger refactoring

Reply:
Ok, so I'll leave it in `LoanPolicy` for now.